library/src/main/scala/scala/scalajs/js/annotation/JSExportStatic.scala

```diff
@@ -23,7 +23,6 @@ import scala.annotation.meta._
  *
  *  @see [[https://www.scala-js.org/doc/interoperability/sjs-defined-js-classes.html Write JavaScript classes in Scala.js]]
  */
-@field @getter @setter
```

Comment:
You're right, it shouldn't have been there in the first place.

It's important *not* to have these meta-annotations so that an annotation on a `val` or `var` is applied *only* on the field, and not on its getter and setter.